### PR TITLE
Update OWNERS with IBM folks

### PR DESCRIPTION
**Pull request by heyselbi**
Update Owners file with IBM folks

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,17 +1,25 @@
 approvers:
   - dtrifiro
+  - fialhocoelho
   - heyselbi
-  - rpancham
+  - joerunde
```
Comment (dtrifiro) [inline OWNERS:5]: @z103cb raises a good point, Nick Hill ([njhill](https://github.com/njhill)) is missing

Comment (prashantgupta24) [reply OWNERS:5]: 👍 yeah how did we forget him

**Comment by heyselbi** [comment]
A separate IBM group/team needs to be created in ODH for this PR to be merged - only ODH owners can do it. Taneem is on PTO right now, but I am looking for someone else to get it created. Hold on tight!

File: OWNERS (hunk 1 above)
Comment (dtrifiro) [reply OWNERS:5]: @heyselbi would you mind adding Nick so we can merge this?

**Comment by dtrifiro** [comment]
Superseded by https://github.com/opendatahub-io/vllm/pull/89

**Pull request closed without merging**